api/meetups/tests/test_meetups.py

```diff
@@ -307,3 +318,95 @@ def test_invalid_image_url(self) -> None:
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertEqual(response.data[0].get(
             'error'), 'InvalidUrl is not a valid image url')
+
+
+class TestRsvpModel(TestCase):
+    """
+    Test for the Rsvp model
+    """
+
+    def setUp(self):
+        """
+        Setting up
+        """
+        self.client = APIClient()
+        self.user = User.objects._create_user(
+            name="Escobar Miguel",
+            email="admin@questioner.com",
+            password="@Escobar123"
+        )
+        self.user.is_active = True
+        self.user.save()
+
+        self.admin = User.objects._create_user(
+            name="Admin User",
+            email="admin@help.com",
+            password="@Admin123"
+        )
+        self.user.is_active = True
+        self.is_admin = True
+        self.user.save()
+
+        self.meetup = Meetup.objects.create(
+            title='Test Driven Development',
+            body='Developers need to discusss the correct approach of doing test driven development',
+            location='Andela Campus',
+            creator=self.admin,
+            scheduled_date=timezone.now() + timezone.timedelta(days=3)
+        )
+        self.meetup.save()
+
+    def login_user(self, email="admin@questioner.com", password="@Escobar123"):
+        """
+        Login in a user to get the token
+        """
+        url = reverse("user_login")
+        data = self.client.post(
+            url,
+            data=json.dumps({
+                "email": email,
+                "password": password
+            }),
+            content_type="application/json"
+        )
+        res = data.data['token']
+
+        return res
+
+    def is_authenticated(self):
+        """
+        Authenticate user before posting data
+        """
+        token, created = Token.objects.get_or_create(user=self.user)
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+
+    def test_post_rsvp(self):
+        """
+        Post rsvp specific to a meetup
+        """
+        meetup_id = str(self.meetup.id)
+        url = reverse('rsvp', args=[meetup_id])
+        response = self.client.post(
+            url,
+            data=json.dumps({
+                "response": "Yes"
+            }),
+            content_type="application/json"
+        )
+
+        return response
+
+    def post_rsvp_without_response(self):
```

Comment:
The test method should start with the key word test like `test_post_rsvp_without_response(self)`